admin_guide/manage_users.adoc

```diff
@@ -9,84 +9,77 @@
 each user].
 
+[[managing-users-deleting-users]]
 == Deleting Users
 
+This section reviews how to properly delete a user account and explains reasons
+for common errors in user account deletion.
+
+[[manging-users-improper-deletion]]
 === Improper Deletion
 
-OpenShift user configuration is stored in several locations within OpenShift.
-Regardless of the identity provider, OpenShift internally stores details like
-role-based access control (RBAC) information and group membership. To completely
-remove user information, this data must be removed in addition to the user
-account.
+{product-title} user configuration is stored in several locations within
+{product-title}. Regardless of the identity provider, {product-title} internally
+stores details like role-based access control (RBAC) information and group
+membership. To completely remove user information, this data must be removed in
+addition to the user account.
 
-If an account is improperly deleted, the following type of error can be seen,
-even though the identification provider already shows the user *myuser* as
-existing:
+If an account is improperly deleted, the following type of error can be seen by
+the user, even though the identification provider already shows the user
+*myuser* as existing:
 
 ====
 ----
-{
     "MESSAGE": "E0412 06:07:48.142862     465 login.go:141] Unable to authenticate password: Error creating or updating mapping for: &api.DefaultUserIdentityInfo{ProviderName:\"htpasswd_auth\", ProviderUserName:\"myuser\", Extra:map[string]string{}} due to users \"myuser\" not found",
-    "PRIORITY": "6",
-    "SYSLOG_FACILITY": "3",
-    "SYSLOG_IDENTIFIER": "atomic-openshift-master",
-    "_BOOT_ID": "18e98783c33546b4a4346f6750ef84a5",
-    "_CAP_EFFECTIVE": "1fffffffff",
-    "_CMDLINE": "/usr/bin/openshift start master --config=/etc/origin/master/master-config.yaml --loglevel=2",
-    "_COMM": "openshift",
-    "_EXE": "/usr/bin/openshift",
-    "_GID": "0",
-    "_HOSTNAME": "master.example.com",
-    "_MACHINE_ID": "f9370ed252a14f73b014c1301a9b6d1b",
-    "_PID": "465",
-    "_SELINUX_CONTEXT": "system_u:system_r:init_t:s0",
-    "_SYSTEMD_CGROUP": "/system.slice/atomic-openshift-master.service",
-    "_SYSTEMD_SLICE": "system.slice",
-    "_SYSTEMD_UNIT": "atomic-openshift-master.service",
-    "_TRANSPORT": "stdout",
-    "_UID": "0",
-}
 ----
 ====
 
-This error is generated because OpenShift sees the user data in the internal
-database that does not correspond to the external identity.
+This error is generated because {product-title} sees the user data in the
+internal database that does not correspond to the external identity.
 
-In OpenShift, two object types contain user data outside the identification
-provider: type user and type identity.
+In {product-title}, two object types contain user data outside the
+identification provider: `*user*` and `*identity*`.
 
-Type user:
+To get the current list of users:
 
 ====
 ----
-$ oc get user ashley
+$ oc get user demo
```

Comment:
@ahardin-rh Since we say this is to get the current "list", we can drop "demo" from the command. As it is now, it would only return the entry for "demo".